### PR TITLE
Tidy up the start of the using draupnir page.

### PR DESCRIPTION
**Pull request by Gnuxie**
Formalize and clarify some of the language, and remove some ambiguities.

Just a general tidy up really.

### CODE REVIEW

File: content/docs/how_to_use_draupnir.md [hunk 1]
```diff
@@ -31,50 +36,65 @@ You can only protect rooms you have admin access to. Abusing the bot for spam wi
...
+subscribe to moderation actions.
+In the most basic setup, policy lists can be thought of as a database of all your bans for your bot.
+However in a more advanced setup, policy lists allow you
+to share moderation decisions with other communities.
+This includes watching the lists from other communities in Matrix or
+collaborating with them on the same lists.
 
 First of all you should define a default policy list. This is where all your actions go to.
 
+{{/* FIXME: why are we doing this? doesn't d4all create them a list called `list`? */}}
```
Comment (Gnuxie) [inline content/docs/how_to_use_draupnir.md:77]: This is a problem.

Comment (MTRNord) [reply content/docs/how_to_use_draupnir.md:77]: I guess then we should omit it or say its for additional lists? :thinking:

**Review by MTRNord — CHANGES_REQUESTED** [review]
meow meow (see your comment)

**Review by MTRNord — APPROVED** [review]
LGTM

**Pull request merged**